velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -41,6 +43,252 @@ class RegexFunctionsTest : public test::SparkFunctionBaseTest {
     return evaluateOnce<std::string>(
         fmt::format("regexp_extract(c0, '{}')", pattern), str);
   }
+
+  void regex_replace_all_params_constant(
```

Comment:
Please, review coding style guidelines and update to comply.

Reply:
Updated. I believe they comply now